indico/modules/events/registration/client/js/form_submission/RegistrationFormSubmission.jsx

```diff
@@ -37,6 +39,34 @@ import {
 
 import '../../styles/regform.module.scss';
 
+function PrivacyPolicy({url}) {
+  const label = (
+    <label>
+      <Translate>
+        I have read and agree to the{' '}
+        <Param name="url" wrapper={<a href={url} target="_blank" rel="noreferrer" />}>
+          Privacy policy
+        </Param>
+      </Translate>
+    </label>
+  );
+
+  return (
+    <Message info style={{marginTop: 25}}>
+      <Message.Header>
+        <Translate>Privacy policy</Translate>
+      </Message.Header>
+      <Form as="div" style={{marginTop: 10}}>
+        <FinalCheckbox required label={label} name="agreed_to_privacy_policy" toggle />
```

Comment:
When this checkbox is selected the submission fails because the schema on the server does not expect the `agreed_to_privacy_policy` field... so it needs to be included in the schema as well

Reply:
It's strange that I didn't run into this issue when I was testing (though it's been a while..) The fix is to just drop the value from the form data since there's no need to store this in the backend at all - it's not possible to submit the form w/o accepting the privacy policy so any existing registration will have accepted it

Reply:
Sending it to the backend and only discarding it there would have the advantage to also catch anything registering by calling the API directly. But since this is an internal API I don't think that's necessary, so your fix is fine